nc_test4/tst_empty_vlen_unlim.c

```diff
@@ -112,18 +113,17 @@ int main() {
     data[2].p = dat2;
     data[2].len = VLEN2;
 
-    //printf("\t* Puting data in VLEN variable:\tnc_put_vara().\n");
-    //stat = nc_put_vara(ncid,varid,&startp,&countp,data);
-    //stat = nc_put_var(ncid,varid,&data);
-    //if(stat) ERR;
-
-
+    printf("\t* Puting data in VLEN variable:\tnc_put_vara().\n");
+    stat = nc_put_vara(ncid,varid,startp,countp,data);
```

Comment:
Uncommenting these lines, and the similar ones below, caused a bunch of the warnings to go away.